admin/config.search-indexes-dump.sh

```diff
@@ -0,0 +1,5 @@
+RSYNC_FULLEXPORT_HOST='10.2.2.40'
+RSYNC_FULLEXPORT_DIR="$FTP_DATA_DIR/search-indexes"
```

Comment:
Shouldn't `SEARCH_INDEX_DUMP_DIR` be set to here, not /home/musicbrainz/search-index-dumps? 'Cause right now admin/RunSearchIndexesDump dumps to `$SEARCH_INDEX_DUMP_DIR`, but then bin/rsync-fullexport-files is set to copy files from here, which would be empty unless I'm missing something.

Reply:
Or alternatively, the other way around: set `$RSYNC_FULLEXPORT_DIR` to `$SEARCH_INDEX_DUMP_DIR` or some subdirectory thereof. I can't find where it copies anything to `$FTP_DATA_DIR` at least.

Reply:
Right, my bad, fixed in https://github.com/metabrainz/musicbrainz-server/pull/1407/commits/b326eb240c2abff6ccbc72eb81ced42ee7c13db1.

Also fixed rsync remote host in https://github.com/metabrainz/musicbrainz-server/pull/1407/commits/bd118dd1b8f9c94254a1e65e8dc99e80dbb3f0ec as it doesn’t need to be different from the default (williams, also the target for deployment which has enought free disk space).